clang-plugin/MozsearchIndexer.cpp

```diff
@@ -2501,7 +2512,7 @@ class IndexConsumer : public ASTConsumer,
     if (SyntaxKind) {
       std::string Mangled = getMangledName(CurMangleContext, ND);
       visitIdentifier("use", SyntaxKind, getQualifiedName(ND), Loc, Mangled,
-                      MaybeType, getContext(SpellingLoc));
+                      MaybeType, getContext(SpellingLoc), Heuristic);
```

Comment:
nit: the comment above this function can be updated from "These should be distinguished from concrete results ... once bug 1833552 is fixed" to "These are distinguished from concrete results ..."